turbo.json

```diff
@@ -10,10 +10,15 @@
         "storybook-static/**",
         ".contentlayer/**"
       ],
-      "dependsOn": ["@kampus/prisma#prisma:generate", "^build"]
```

Comment:
[NIT] for some reason your prettier setup doesn't work on this file :D

Reply:
my prettier works different on json i guess let me check vscode settings 👍